node_package/tests/ReactOnRails.test.js

```diff
@@ -21,7 +21,7 @@ test('ReactOnRails render returns a virtual DOM element for component', (assert)
   ReactOnRails.register({ R1 });
 
   // eslint-disable-next-line no-underscore-dangle
-  const actual = ReactOnRails.render('R1', {}, 'root')._reactInternalInstance._currentElement.type;
+  const actual = ReactOnRails.render('R1', {}, 'root')._reactInternalFiber.type;
```

Comment:
```
// When updating to React v16 there was a breaking change 
// that caused the below test to fail
test('ReactOnRails render returns a virtual DOM element for component', (assert) => {
  assert.plan(1);
  const R1 = createReactClass({
    render() {
      return (
        <div> WORLD </div>
      );
    },
  });
  ReactOnRails.register({ R1 });
  // eslint-disable-next-line no-underscore-dangle
  const actual = ReactOnRails.render('R1', {}, 'root')._reactInternalInstance._currentElement.type;
  assert.deepEqual(actual, R1,
    'ReactOnRails render should return a virtual DOM element for component');
});
// The error occurred because _reactInternalInstance was deprecated in React 16
// The object returned from ReactDOM.render() for v15.6 looked like this (simplified):
{
    _reactInternalInstance: {
        _currentElement: {
            type: // Stuff here
        }
    }
}
// After upgrading to React v16, the object looked like this (again, simplified):
{
    _reactInternalFiber: {
        type: // Stuff here
    },
    _reactInternalInstance: {
        // EMPTY!
    }
}
// A one line update to the test above resulted in all tests passing
const actual = ReactOnRails.render('R1', {}, 'root')._reactInternalFiber.type;
```